Gemfile.lock

```diff
@@ -123,12 +124,16 @@ GEM
     minitest (5.11.3)
     mongo (2.6.2)
       bson (>= 4.3.0, < 5.0.0)
-    mongoid (7.0.2)
-      activemodel (>= 5.1, < 6.0.0)
+    mongoid (5.4.0)
+      activemodel (~> 4.0)
```

Comment:
interesting, this goes back 2 versions - blocker?

Reply:
Yes for sure. Don’t downgrade mongoid. Something’s off.

Reply:
Just revert `Gemfile.lock` and bundle without a bundle update.

Reply:
Looks like mongoid-enum hasn't been maintained.

I had opened https://github.com/thetron/mongoid-enum/issues/39 a while ago. Emailed the maintainer now.